hangman.py

```diff
@@ -0,0 +1,240 @@
+    returns: string, comprised of letters, underscores (_), and spaces that represents
+      which letters in secret_word have been guessed so far.
+    '''
+    # FILL IN YOUR CODE HERE AND DELETE "pass"
+    pass
+
+
+def get_available_letters(letters_guessed):
+    '''
+    letters_guessed: list (of letters), which letters have been guessed so far
+    returns: string (of letters), comprised of letters that represents which letters have not
+      yet been guessed.
+    '''
+    # FILL IN YOUR CODE HERE AND DELETE "pass"
+    pass
+
+
+def warnings(value_mistake, warnings_point, guesses_point):
+    """
+    Subtract warnings point and output name of error
+    """
+
+    if warnings_point > 0:
+        warnings_point -= 1
+        print(f"Oops! {value_mistake}")
+        print("You have", warnings_point, "warnings left:", end=" ")
+    else:
+        guesses_point -= 1
+        print("Oops! You have no warnings left, so you lose one guess:", end=' ')
+    return warnings_point, guesses_point
+
+
+def check_star_30(count, secret_word, letters_guessed):
+    pass
+
+
+def letter_in_word(secret_word, letter, letter_guessed, guesses_point, letters_guessed):
+    """
+    Check if letter is word, else subtracts point
+    """
+
+    letters_guessed.add(letter)
+    if letter in secret_word:
+        print("Good guess:", get_guessed_word(secret_word, letters_guessed))
+    else:
+        if letter in LIST_VOWELS:
+            guesses_point -= 2
+        else:
+            guesses_point -= 1
+        print("Oops! That letter is not in my word:", get_guessed_word(secret_word, letters_guessed))
+    return letters_guessed, guesses_point
+
+
+def classic_input(count_star, warnings_point, guesses_point):
+    if count_star != 0:
+        warnings_point, guesses_point = warnings("you are playing classic hangman", warnings_point, guesses_point)
+    else:
+        print("Oops, you are playing classic hangman:", end=" ")
+        count_star += 1
+    return warnings_point, guesses_point, count_star
+
+
+def hangman(secret_word, request):
+    '''
+    Game Hangman. "request" show hangman(classic) or hangman with hints.
+    Print information for user, start loop, check user choose hangman or hangman with hints,
+    Check input letter and call letter_in_word, check if user guessed word.
+
+    count_star is variable, which count all input star. It need if user play in hangman with hints and input star more
+    than 30 times, program output random possible words.
+    '''
+
+    count_star = 0
+    guesses_point = 6
+    warnings_point = 3
+    letters_guessed = set()
+    print("I am thinking of a word that is", len(secret_word), "letters long")
+    print("You have", warnings_point, "warnings left")
+
+    while guesses_point > 0:
+        print("-------")
+        print("You have", guesses_point, "guesses left")
+        print("Available letters:", end=" ")
+        get_available_letters(letters_guessed)
```

Comment:
`get_available_letters` should return a string, which you need to print, so you can combine these lines I believe